src/Campaigns/Actions/RegisterCampaignPagePostType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Give\Campaigns\Actions;
+
+/**
+ * @unreleased
+ */
+class RegisterCampaignPagePostType
+{
+    /**
+     * @unreleased
+     */
+    public function __invoke()
+    {
+        register_post_type( 'give_campaign_page', [
+            'label' => __('Campaign Page', 'give-peer-to-peer'),
```

Comment:
Let's update this textdomain

Reply:
Resolved, see [466f978](https://github.com/impress-org/givewp/pull/7510/commits/466f978ca0313b47cd25a11d4c387527f8320cd9).